processing/bloom_handler.go

```diff
@@ -156,7 +169,15 @@ func MakeBloomHandlerFromFile(bloomFilename string, compressed bool,
 			return nil, err
 		}
 	}
+	log.WithFields(log.Fields{
+		"domain": "bloom",
+	}).Infof("loading Bloom filter '%s'", bloomFilename)
```

Comment:
May want to move this up towards beginning of the function, before any error path